app/lib/analyzer/task_sources.dart

```diff
@@ -4,24 +4,79 @@
 
 import 'dart:async';
 
+import 'package:gcloud/db.dart';
+
+import '../frontend/models.dart';
 import '../shared/task_scheduler.dart';
 
+import 'models.dart';
+
+/// Creates a task when a package has no analysis for its latest stable version.
 class DatastoreHeadTaskSource extends PollingTaskSource {
-  DatastoreHeadTaskSource() : super(new Duration(minutes: 10));
+  final DatastoreDB _db;
+  DatastoreHeadTaskSource(this._db) : super(new Duration(minutes: 10));
 
   @override
   Stream<Task> pollTasks() {
-    // TODO: implement pollTasks
-    return new Stream.empty();
+    return _db
+        .query(Package)
```

Comment:
> the history poller is just going back in history once and then it's done

There is a conflicting requirement here: we want to re-analyze packages with their latest resolved dependencies, and that will require polling history again and again (at least until we have a better way of detecting the need of re-analysis).